depinject/appconfig/module.go

```diff
@@ -24,6 +24,38 @@ var Register = RegisterModule
 // Protobuf message types used for module configuration should define the
 // cosmos.app.v1alpha.module option and must explicitly specify go_package
 // to make debugging easier for users.
+//
+// If you want to customize an existing module, you need to overwrite by calling RegisterModule
```

Comment:
I added a bit of the same line 20. Could you maybe delete it then or move it down? Otherwise that's a bit repetitive.

Reply:
I've updated the previous section.